backend/src/routes/api/quickstarts/list.ts

```diff
@@ -1,6 +1,9 @@
 import { QuickStart } from '../../../types';
 import { getQuickStarts } from '../../../utils/resourceUtils';
+import { checkJupyterEnabled } from '../../../utils/componentUtils';
 
 export const listQuickStarts = (): Promise<QuickStart[]> => {
-  return Promise.resolve(getQuickStarts());
+  return Promise.resolve(
+    getQuickStarts().filter((qs) => checkJupyterEnabled() || qs.spec.appName !== 'jupyter'),
```

Comment:
I think you want this:

```suggestion
    getQuickStarts().filter((qs) => (checkJupyterEnabled() ? qs.spec.appName !== 'jupyter' : true)),
```

Your logic is hiding away all jupyter resources no matter what

Reply:
No, I don't think that's the case. If Jupyter is enabled, `checkJupyterEnabled()` will return true, and the whole statement will be `true`, and everything will be returned, nothing will be filtered out. If Jupyter is disabled, then it will go to check `qs.spec.appName !== 'jupyter'` and filter out all the Jupyter related quick starts.

Reply:
My bad! Yes you are right, damn I misread that, resolving this conversation